### ISSUE TITLE
Move visualization of GP output to a separate file

### ISSUE THREAD

**Issue by acxz** [acxz/pl-utils#8]
- [x] Figure out how to load trained GP model checkpoint (need to make sure the original training data is saved with the rest of the model hyperparameters)

- [ ] This can also involve saving as an onnx file and loading it with tensorrt as in a production environment.

- [x] I think to do this I need to first revamp how I am using hparams: See: https://pytorch-lightning.readthedocs.io/en/stable/hyperparameters.html?highlight=hyperparameter

**Comment by acxz** [comment]
Also adding a 2D visualization and 3D input-output case would be sooo cool. See tubes now thats cray cray

**Comment by acxz** [comment]
Something important to visualize is the overlayed Gaussians on the z axis at every point for the confidence region.
